presto-hive/src/test/java/com/facebook/presto/hive/TestParquetDistributedQueries.java

```diff
@@ -93,6 +93,31 @@ public void testSubfieldWithSchemaChanges()
         }
     }
 
+    @Test
+    public void testSubfieldWithSchemaChanges2()
```

Comment:
@vkorukanti could we merge this into testSubfieldWithSchemaChanges? or have a new name? testSubfieldWithSchemaChanges2 seems bad naming

Reply:
done.